akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package jdocs.akka.typed;
+/*
+ * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import akka.actor.typed.ActorSystem;
+// #pool
+import akka.actor.typed.ActorRef;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.GroupRouter;
+import akka.actor.typed.javadsl.PoolRouter;
+import akka.actor.typed.javadsl.Routers;
+import akka.actor.typed.receptionist.Receptionist;
+import akka.actor.typed.receptionist.ServiceKey;
+
+// #pool
+
+public class RouterTest {
+
+  static // #pool
+  class Worker {
+    interface Command {}
+
+    static class DoLog implements Command {
+      public final String text;
+
+      public DoLog(String text) {
+        this.text = text;
+      }
+    }
+
+    static final Behavior<Command> behavior =
+        Behaviors.setup(
```

Comment:
maybe include restart supervision here, since that is probably what you want for pools

Reply:
Good idea.